modules/api/php/endpoints/candidate/visit/instrument/instrument.class.inc

```diff
@@ -92,6 +92,13 @@ class Instrument extends Endpoint implements \LORIS\Middleware\ETagCalculator
     {
         $pathparts = $request->getAttribute('pathparts');
         $flags     = array_search('flags', $pathparts) !== false;
+        $user      = $request->getAttribute('user');
+
+        if ($this->_instrument->_hasAccess($user)) {
+            return new \LORIS\Http\Response\JSON\Forbidden(
+                'Can not update instruments for user' . $user
```

Comment:
I don't think `$user` is a string.

Reply:
`_hasAccess` does not take a string https://github.com/aces/Loris/blob/c1bb1daeed25cde5e258db626b02c17395bc14d9/php/libraries/NDB_BVL_Instrument.class.inc#L302

Reply:
nevermind I just got it

Reply:
done